### DISCUSSION TITLE
Clockwork V2 Devlog - April 28

### DISCUSSION THREAD

**Discussion by ToasterPanic** [red-stone-network/clockwork#43] [Announcements]
Hello! I'm gonna start making devlogs for Clockwork V2 to help y'all keep up with the latest added features.

### What's new in Clockwork V2
We're gonna start off with the new notification system! Apps can now send notifications to Clockwork using clockwork.js.
![fart](https://user-images.githubusercontent.com/91223726/235201970-db5beda3-df62-4fdb-9097-e3c190b55ed2.gif)

We also have upgraded the app store - it now has an account system and app uploading!

There's also a passcode system, which helps keep people from touching your Clockwork without permission.
![ttt](https://user-images.githubusercontent.com/91223726/235207253-a3f2450d-6b83-4c5d-870d-c87697e3f705.gif)

Thanks for reading!

**Comment by JacoobK** [comment]
Where is the discord, I want to see what websites work with the bookmarklet but I can’t find a link

**Reply by ToasterPanic** [reply]
I can just tell you here: example.com, clever.com, maybe ixl.com. Any website with little / no content security policies will work.
